app/src/main/java/org/fossasia/openevent/app/core/event/about/AboutEventFragment.java

```diff
@@ -125,11 +125,13 @@ public boolean onOptionsItemSelected(MenuItem item) {
             case R.id.action_create_change_copyright:
                 if (creatingCopyright) {
                     getFragmentManager().beginTransaction()
-                        .replace(R.id.fragment, CreateCopyrightFragment.newInstance())
+                        .add(R.id.fragment, CreateCopyrightFragment.newInstance())
```

Comment:
Don't use add

Reply:
Using `#add` will retain AboutEventFragment's state. So on back-press, no progress bar will be shown, making it look smooth. The newly created copyright is added instantly to event details without a progress bar. What do you think?

Reply:
It takes more memory but alright, we'll switch to add if there are more fragments

Reply:
Is that a yes or do we add `add` later?